recipes/xsv/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.13.0" %}
+
+package:
+  name: xsv
+  version: {{ version }}
+
+source:
+  url: https://github.com/BurntSushi/xsv/archive/{{ version }}.tar.gz
+  sha256: 2b75309b764c9f2f3fdc1dd31eeea5a74498f7da21ae757b3ffd6fd537ec5345
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  host:
+    - rust
+
+test:
+  commands:
+    - xsv --help > /dev/null
+
+about:
+  home: https://github.com/BurntSushi/xsv
+  license: MIT
```

Comment:
If you're going to distribute this under the `MIT` license, please include the `license_file`, which should be in the source distribution.

Reply:
alright I will add it.